frameworks/hdfs/src/main/dist/core-site.xml

```diff
@@ -656,5 +630,41 @@
         <name>hadoop.security.authorization</name>
         <value>true</value>
     </property>
-    {{/SECURE_MODE}}
+    <property>
+        <name>hadoop.rpc.protection</name>
+        <value>privacy</value>
+    </property>
+    <property>
+        <name>hadoop.security.auth_to_local</name>
+        <value>
+          RULE:[2:$1@$0](.*)s/.*/{{TASK_USER}}/
```

Comment:
{TASK_USER} does this resolves to "hdfs" user?

Reply:
No, this resolves to `nobody` - it was just a placeholder so we can discuss it.